src/js/console.js

```diff
@@ -11,45 +19,102 @@ function getConsoleLogs() {
   );
 }
 
-function formattedMessages(messages) {
-  const formattedMessages = messages.map(message => {
-    if (typeof message === 'string') {
-      return message;
-    } else if (message.stack) {
-      return message.stack;
-    } else if (typeof message === 'object') {
-      return JSON.stringify(message);
-    }
+/* function formattedMessages(messages) {
+ const formattedMessages = messages.map(message => {
+   if (typeof message === 'string') {
+     return message;
+   } else if (message.stack) {
+     return message.stack;
+   } else if (typeof message === 'object') {
+     return JSON.stringify(message);
+   }
+   return message;
+ });
+ return formattedMessages;
+}; */
+function formattedMessages(message) {
+  if (typeof message === 'string') {
     return message;
-  });
-  return formattedMessages;
+  }
+  else if (typeof message === 'object') {
+    return JSON.stringify(message);
+  }
 };
 
-console.log = function() {
+console.log = function (arguments) {
   const consoleLogs = getConsoleLogs();
-  consoleLogs.push({ dateTime: new Date(), type: 'log', message: formattedMessages(Array.from(arguments)) });
+  consoleLogs.push({ dateTime: getPSTFromUTC(new Date()), type: 'log', message: formattedMessages(arguments) });
```

Comment:
Why arguments are not converted to array only in this method

Reply:
it returns array of characters